### ISSUE TITLE
Proper exporting when using rollup

### ISSUE THREAD

**Issue by jsantell** [google-ar/three.ar.js#63]
When using rollup and consuming three.ar.js as a dependency, we encounter the named export error [reported by rollup](https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module) with a workaround.

```
[!] Error: 'ARUtils' is not exported by node_modules/three.ar.js/dist/three.ar.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module

`import { ARUtils, ARView, ARPerspectiveCamera } from 'three.ar.js';`
```

We should fix this on our end for the umd-style build that we have currently, and may include also having an es6 module distributable for this purpose.

**Comment by jsantell** [comment]
This might just be what happens when consuming UMD -- publishing a module format (and package.json module entry) would solve this, I think

**Comment by jsantell** [comment]
Fixed in #89
